libs/components/behavioral/src/lib/contact-behavior/contact-behavior.component.scss

```diff
@@ -1,3 +1,13 @@
 :host {
   display: block;
+
+  .contact-modal {
```

Comment:
Tweak the min/max width for different screen sizes.
Also add scrolling when min width/height is reached